src/test/java/ToDoTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void Test(){
```

Comment:
take note of the naming convention here. It seems more like you are testing toFormattedString() then the constructor.